packages/cli/test/integration/workflows/workflows.controller.ee.test.ts

```diff
@@ -527,6 +527,35 @@ describe('GET /workflows/:workflowId', () => {
 		]);
 		expect(member2Workflow.sharedWithProjects).toHaveLength(1);
 	});
+
+	test('should return workflow credentials home project and shared with projects', async () => {
+		const savedCredential = await saveCredential(randomCredentialPayload(), { user: member });
+		// Both users have access to the credential (none is owner)
```

Comment:
```suggestion
		// Both users have access to the credential (none is global owner)
```

Maybe? Just differentiating from owning the cred, which `member` does.